Framework/DataHandling/src/SaveNXcanSAS.cpp

```diff
@@ -384,15 +384,6 @@ void addProcess(H5::Group &group, const Mantid::API::MatrixWorkspace_sptr &works
   Mantid::DataHandling::H5Util::write(process, sasProcessTermCan, std::to_string(canRun));
 }
 
-/**
- * Create a note class within process
- * @param group: the sasEntry
- */
-void createNote(H5::Group &group) {
-  auto process = group.openGroup(sasProcessGroupName);
-  Mantid::DataHandling::H5Util::createGroupCanSAS(process, sasNoteGroupName, nxNoteClassAttr, sasNoteClassAttr);
-}
-
 /**
  * Add a note containing sample or can run numbers to the process group.
```

Comment:
There are a few comments around and within this method referring to notes that might be worth updating now that there isn't a notes section.